bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/common/registry/Registry.java

```diff
@@ -49,15 +54,15 @@
      *            key of the element
      * @return element or null if no element was found
      */
-    public E get(K key);
+    public @Nullable E get(K key);
```

Comment:
My concerns was that exactly this was said in the javadoc (but in case if it is not readed ok) and it is obvious for me that it will return a null for any key which without value (as a Map).